app/helpers/petition_helper.rb

```diff
@@ -0,0 +1,32 @@
+module PetitionHelper
+  def render_petition_form(petition, form)
+    [
+      render_hidden_details(petition, form),
+      render_ui(petition, form)
+    ].join.html_safe
```

Comment:
Ah, yes. I tried `capture` but forgot that I needed `concat` too to make it work.  Would probably make the ``render_hidden_details`` method clearer too.  Thanks!